app/components/govuk_component/start_button_component.rb

```diff
@@ -1,4 +1,10 @@
 class GovukComponent::StartButtonComponent < GovukComponent::Base
+  BUTTON_ATTRIBUTES = {
+    role: 'button',
+    draggable: 'false',
+    data: { module: 'govuk-button' }
+  }
```

Comment:
Looks great overall, the only bit I'm not sure on is where `BUTTON_ATTRIBUTES` should live if it's shared between components and helpers.